pylearn2/models/tests/test_mlp.py

```diff
@@ -435,22 +435,33 @@ def test_get_layer_monitor_channels():
                                       state=None, targets=targets)
 
 
-def test_flattener_layer():
+# Compares two mlp with a single mlp composed of the two mlps
+# and a flattener layer
```

Comment:
It does what? Which is nested mlp and which is not? 

Please get this comment under function signature and format everything as a docstring like you did [here](https://github.com/lisa-lab/pylearn2/pull/1444/files#diff-102b66694f1d64cec964b556c0a33dd8R638)